legacy/src/guides/merchant-terminals.md

```diff
@@ -1,146 +1,4 @@
 ---
-layout: default
-parent: Guides
-nav_order: 2
-title: Merchant Terminals
 permalink: /guides/merchant-terminals
+redirect_to: /guides/payment-flows
```

Comment:
@IsaacMurtagh Can we handle this redirect with Astro? Ultimately, we want to get rid of this `legacy` folder

Reply:
@tom-blake, this can only be done in astro if we use SSR.
SGS give better performance as they're cached, so it may be worth investigating if we can serve only some sites using SSR. I'll split that into a new PR